ui/page/dcrdex/dcrdex_page.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/crypto-power/cryptopower/app"
 	"github.com/crypto-power/cryptopower/ui/cryptomaterial"
 	"github.com/crypto-power/cryptopower/ui/load"
+	"github.com/crypto-power/cryptopower/ui/page/components"
+	"github.com/crypto-power/cryptopower/ui/values"
 )
 
 const DCRDEXID = "DCRDEXID"
```

Comment:
Not added by your PR, but `DCRDEXPageID`? I don't know what `DCRDEXID` mean.

Reply:
Changed to DCRDEXPageID